compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/model/symbols/Symbols.java

```diff
@@ -223,9 +223,7 @@ public static BXMLNSSymbol createXMLNSSymbol(Name name,
     }
 
     public static BVarSymbol createVarSymbolForTupleMember(BType type) {
-        BTypeSymbol tsymbol = type.tsymbol;
-        return new BVarSymbol(type.flags, tsymbol.name, tsymbol.pkgID, type, tsymbol.owner,
-                tsymbol.pos, tsymbol.origin);
+        return new BVarSymbol(null, null, null, type, null, null, null);
```

Comment:
Does this compile? :)

```suggestion
        return new BVarSymbol(0, null, null, type, null, null, null);
```

Reply:
My bad, fixed it.